modules/structs/hook.go

```diff
@@ -494,3 +487,27 @@ type PackagePayload struct {
 func (p *PackagePayload) JSONPayload() ([]byte, error) {
 	return json.MarshalIndent(p, "", "  ")
 }
+
+// CommitStatusPayload represents a payload information of commit status event.
+type CommitStatusPayload struct {
+	// TODO: add Branches
+	Commit  *PayloadCommit `json:"commit"`
+	Context string         `json:"context"`
+	// swagger:strfmt date-time
+	CreatedAt   time.Time `json:"created_at"`
+	Description string    `json:"description"`
+	ID          int64     `json:"id"`
+	// Name        string         `json:"name"`
```

Comment:
?

Reply:
Still, why is the `Name` commented out?

Reply:
9363b99f1ba24e3bbf0b34da44cba1025bfdabeb